apispec/openapi.json

```diff
@@ -5280,7 +5280,7 @@
       "transport.status": {
         "type": "string",
         "title": "status",
-        "description": "Indicates the status of this transport, it must be disabled or enabled. Default value is enabled. If status is disabled, this transport is invalid. ",
+        "description": "Indicates the status of this transport. It must be disabled or enabled. The default value is enabled. If the status is disabled, this transport is invalid. ",
```

Comment:
Suggestion to replace the first two phrases with:  "Indicates the status of this transport, either disabled or enabled."

Reply:
I agree with Burton.